keras/layers/local.py

```diff
@@ -41,7 +41,7 @@ class LocallyConnected1D(Layer):
             specifying the stride length of the convolution.
             Specifying any stride value != 1 is incompatible with specifying
             any `dilation_rate` value != 1.
-        padding: One of `"valid"` or `"same"` (case-insensitive).
+        padding: `"valid"` (case-insensitive).
```

Comment:
If there is only one possible value then there shouldn't be an argument. Removing the argument would cause further backwards compatibility issues, so instead please just add a sentence explaining that more values could be added in the future but that only one value is supported for now.